docs/index.md

```diff
@@ -105,6 +108,18 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional) Default Enterprise Project ID for supported resources.
   If omitted, the `G42_ENTERPRISE_PROJECT_ID` environment variable is used.
 
+* `endpoints` - (Optional) Configuration block in key/value pairs for customizing service endpoints.
+  The following endpoints support to be customized: autoscaling, ecs, ims, vpc, nat, evs, obs, sfs, cce, rds, dds, iam.
```

Comment:
dose g42 supports all the services?

Reply:
Some of these services are not supported for now.
I have fixed this part, only supported services are shown in the doc.